src/main/java/org/jenkinsci/plugins/github_branch_source/Endpoint.java

```diff
@@ -151,15 +143,17 @@ public FormValidation doCheckApiUri(@QueryParameter String apiUri) {
                 GitHub github = GitHub.connectToEnterpriseAnonymously(api.toString());
                 github.checkApiUrlValidity();
                 LOGGER.log(Level.FINE, "Trying to configure a GitHub Enterprise server");
-                // For example: https://api.github.com/ or https://github.mycompany.com/api/v3/ (with private mode disabled).
+                // For example: https://api.github.com/ or https://github.mycompany.com/api/v3/ (with private mode
+                // disabled).
```

Comment:
Not helpful IMO.

Reply:
:shrug: 
Okay, this can be controlled. Line comments to do not join if already wrapped, so user can wrap lines earlier if they choose.